### PR TITLE
Allow to use custom filename && directory name to store configs

### PR DESCRIPTION
**Pull request by feruzoripov**
https://github.com/rubyconfig/config/issues/340

### CODE REVIEW

File: lib/config/version.rb
Comment (pkuczynski) [inline lib/config/version.rb:2]: I would leave this to the maintainers, whenever they would be releasing the gem... Please revert...

**Review by pkuczynski — COMMENTED** [review]
Can you also update the docs please (README)?

File: lib/config/version.rb
Comment (feruzoripov) [reply lib/config/version.rb]: reverted version and updated README @pkuczynski

**Review by cjlarose — APPROVED** [review]

**Comment by cjlarose** [comment]
Released with version 5.2.0

**Pull request merged**
